app/services/ClubManagementService.php

```diff
@@ -99,12 +102,21 @@ static function getSelectedClub()
         return $club;
     }
 
+    public function getSelectedClub()
```

Comment:
This is the same as `getClub`. It might not be needed.

Reply:
Ok with that, thanks

Reply:
Awesome then I will resolve this